Source/WebCore/layout/formattingContexts/inline/InlineLevelBox.h

```diff
@@ -83,9 +83,9 @@ class InlineLevelBox {
     LeadingTrim leadingTrim() const { return m_style.leadingTrim; }
     InlineLayoutUnit inlineBoxContentOffsetForLeadingTrim() const { return m_inlineBoxContentOffsetForLeadingTrim; }
 
-    bool hasAnnotation() const { return hasContent() && m_annotation.has_value(); };
-    std::optional<InlineLayoutUnit> annotationAbove() const { return hasAnnotation() && m_annotation->type == Annotation::Type::Above ? std::make_optional(m_annotation->size) : std::nullopt; }
-    std::optional<InlineLayoutUnit> annotationUnder() const { return hasAnnotation() && m_annotation->type == Annotation::Type::Under ? std::make_optional(m_annotation->size) : std::nullopt; }
+    bool hasAnnotation() const { return (hasContent() || isAtomicInlineLevelBox()) && m_annotation.has_value(); };
+    std::optional<InlineLayoutUnit> annotationAbove() const { return hasAnnotation() ? std::optional { m_annotation->above } : std::nullopt; }
+    std::optional<InlineLayoutUnit> annotationBelow() const { return hasAnnotation() ? std::optional { m_annotation->below } : std::nullopt; }
```

Comment:
```suggestion
    std::optional<InlineLayoutUnit> annotationAbove() const { return hasAnnotation() ? { m_annotation->above } : std::nullopt; }
    std::optional<InlineLayoutUnit> annotationBelow() const { return hasAnnotation() ? { m_annotation->below } : std::nullopt; }
```

Does this work? or even without the brackets?

Reply:
no